src/DynamoCore/DSEngine/CodeCompletionServices.cs

```diff
@@ -127,6 +127,37 @@ internal IEnumerable<CompletionData> SearchCompletions(string stringToComplete,
         }
```

Comment:
While I was looking to see if this method can be merged with `SearchCompletions` above, I noticed that `Guid guid` parameter is not used, perhaps @aparajit-pratap can remove it separately.